data/json/construction.json

```diff
@@ -2309,5 +2309,17 @@
     "pre_note": "Can be deconstructed without tools.",
     "pre_terrain": "t_pit_shallow",
     "post_terrain": "f_firering"
+  },
+ {
+    "type": "construction",
+    "description": "Build Rammed Earth Wall",
+    "category": "CONSTRUCT",
+    "required_skills": [ [ "fabrication", 2 ], [ "survival", 2 ] ],
+    "time": 360,
+    "qualities": [ { "id": "HAMMER", "level": 2 } ],
+    "tools": [ [ "frame_wood_light" ], [ "log" ] ],
+    "components": [ [ [ "material_soil", 240 ] ], [ [ "water", 100 ], [ "water_clean", 100 ] ] ],
```

Comment:
I'd suggest requiring either sand, clay, concrete or lime as a stabilizer as well. 
wikipedia:
> Manufacturing rammed earth involves compressing a damp mixture of earth that has suitable proportions of sand, gravel, clay, and/or an added stabilizer into an externally supported frame or mold, forming either a solid wall or individual blocks. Historically, additives such as lime or animal blood were used to stabilize it.

I'm not sure what the proportion to soil would be, I get the impression not all that much. [this group](https://groups.google.com/forum/#!topic/icomos-isceah/JJFvmBcsL3Q) says 3-12% lime, so you could probably get away with something like 20 units of sand/lime/concrete per wall tile?

Reply:
Okay, just adding that in. Concrete seems like an odd one though, because it's measured in 1l units, and you only need four of those to make a simple concrete wall. Considering the proportions you're suggesting, 1 unit of concrete seems fitting, as it contains over 20 sand. Either that, or we have a separate recipe with more modern tools (cement mixer, sledge hammer, etc) that's somewhat faster to construct due to being able to set a concrete mold for the wall.